.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,28 @@
+# How to Contribute
```

Comment:
we already have CONTRIBUTING.md in the top directory, should we move that under `.github` directory?

Reply:
Same comment as above, I'll just exclude this from the copy